foodhub/entities/views.py

```diff
@@ -37,6 +60,13 @@ class StoreViewSet(viewsets.ModelViewSet):
     authentication_classes = [BasicAuthentication]
     permission_classes = [IsAuthenticated]
 
+    @action(detail=True)
+    def orders(self, request, pk=None):
+        store = Store.objects.get(pk=pk)
```

Comment:
can Model.objects.filter(pk=pk).first() be used, as it can handle if key not present also

Reply:
Yup..that makes sense. Will change it. Thx!